security/pkg/nodeagent/caclient/providers/google/client.go

```diff
@@ -110,3 +122,14 @@ func (cl *googleCAClient) getTLSDialOption() (grpc.DialOption, error) {
 	creds := credentials.NewClientTLSFromCert(pool, "")
 	return grpc.WithTransportCredentials(creds), nil
 }
+
+func parseZone(clusterURL string) string {
+	// input: https://container.googleapis.com/v1/projects/quanlinpilot1/locations/us-central1-c/clusters/mpibetacluster2
+	// output: quanlinpilot1 us-central1-c
+	var rgx = regexp.MustCompile(`.*/projects/(.*)/locations/(.*)/clusters/.*`)
+	rs := rgx.FindStringSubmatch(clusterURL)
+	if len(rs) < 3 {
+		return ""
```

Comment:
What would happen if return "", does the client send CSR without zone specification or does the client skip sending CSR?

Reply:
without zone info.